sdk/tools/hhpcomp/chmc/chm.h

```diff
@@ -60,8 +60,9 @@ typedef long long               Int64;
 typedef unsigned long long      UInt64;
 
 /* x86-64 */
+/* arm64/aarch64 */
 /* Note that these may be appropriate for other 64-bit machines. */
```

Comment:
Suggestion: either leave only `/* x86-64 */` here, or update it with all architectures. I'd vote for leaving only first one here, as the comment below explains "these may be appropriate for other 64-bit machines".